tests/Doctrine/Tests/ORM/Functional/ManyToManyBasicAssociationTest.php

```diff
@@ -309,8 +309,10 @@ public function addCmsUserGblancoWithGroups($groupCount = 1)
      */
     public function testUpdateDeleteSizeSubselectQueries()
     {
-        $this->_em->createQuery("DELETE Doctrine\Tests\Models\CMS\CmsUser u WHERE SIZE(u.groups) = 10")->execute();
-        $this->_em->createQuery("UPDATE Doctrine\Tests\Models\CMS\CmsUser u SET u.status = 'inactive' WHERE SIZE(u.groups) = 10")->execute();
+        $this->_em->createQuery("DELETE " . CmsUser::class . " u WHERE SIZE(u.groups) = 10")->execute();
```

Comment:
Shouldn't we do that in a separate PR? I know I might gone to far with the first commit and maybe I shouldn't have done that 😜